tests/test_core.py

```diff
@@ -423,7 +422,7 @@ def test_table_does_not_exist(engine_testaccount):
     """
     meta = MetaData()
     with pytest.raises(NoSuchTableError):
-        Table("does_not_exist", meta, autoload=True, autoload_with=engine_testaccount)
+        Table("does_not_exist", meta, autoload_with=engine_testaccount)
```

Comment:
`autoload` is no longer a valid argument in 2.0. Code works fine without it in 1.4 as well.